File: leftover.cs
using System;

public class Program
{
	private static double Leftover(double m, double p, double q)
	{
		return m * (100 - p) * (100 - q) / 10000;
	}
	
	public static void Main()
	{
		double[] mpq = Array.ConvertAll(Console.ReadLine().Split(), double.Parse);
		Console.WriteLine(Leftover(mpq[0], mpq[1], mpq[2]));
	}
}

File: leftover.swift
import Foundation

func leftover(m: Double, p: Double, q: Double) -> Double {
    return m * (100 - p) * (100 - q) / 10000
}

let mpq = readLine()!.split(separator: " ").map { Double($0)! }
print(leftover(m: mpq[0], p: mpq[1], q: mpq[2]))